Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/README.md

```diff
@@ -0,0 +1,256 @@
+# Color-Scheme Abstraction for Krypton Toolkit
```

Comment:
Can this also be included in here please? [Krypton-Suite/Standard-Toolkit-Online-Help](https://github.com/Krypton-Suite/Standard-Toolkit-Online-Help)